hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -57,25 +59,30 @@
   private final SharedKeyCredentials sharedKeyCredentials;
   private final String xMsVersion = "2018-11-09";
   private final ExponentialRetryPolicy retryPolicy;
+  private final String accountName;
   private final String filesystem;
+  private final AuthType authType;
```

Comment:
can you move down to line 68 along with the accountName, in the hope it will make merge easier

Reply:
Done